src/biome/text/features.py

```diff
@@ -61,7 +61,10 @@ def config(self) -> Dict:
 
     def to_json(self) -> Dict:
         """Returns the config as dict for the serialized json config file"""
-        return self.to_dict()
+        data = vars(self)
+        data.update(data.pop("extra_params", {}))
+
+        return data
 
     def to_dict(self) -> Dict:
```

Comment:
This is the same as `to_json`, not sure why we have both methods really ... But i would either let `to_json` return `to_dict` or `to_dict` return `to_json`, not implementing the logic twice.

Reply:
only keep `to_json` for allenlp `sanitize` integration